src/App.jsx

```diff
@@ -12,6 +12,21 @@ import {
 import Router from "./Router";
 
 function App() {
+  let helpSteps = [];
+
+  const getHelpSteps = () => {
```

Comment:
could be moved into `HelpContext` 
it allows to propagate needed methods into all components